panel/src/components/View/Menu.vue

```diff
@@ -9,6 +9,7 @@
 		<div class="k-panel-menu-body">
 			<!-- Search button -->
 			<k-button
+				:disabled="!hasSearch"
```

Comment:
Wouldn't it be better to completely remove the button if there's no search?

Reply:
When completely removed, the main menu comes to top. I think that's why Nico left it disable instead of remove.

Reply:
@distantnative does that lead to a weird ui issue? Otherwise it makes a lot of sense if it moves up

Reply:
I found it looked a bit odd so close to the top and not aligned with the header headline. But won't object if you think differently.

Reply:
I checked again and I think it's better if the menu moves up. I've changed it.